Packages/com.unity.inputsystem/InputSystem/Plugins/UI/InputSystemUIInputModule.cs

```diff
@@ -1542,14 +1542,20 @@ private void EnableInputAction(InputActionReference inputActionReference)
             action.Enable();
         }
 
-        private static void DisableInputAction(InputActionReference inputActionReference)
+        private void DisableInputAction(InputActionReference inputActionReference, bool onDisableComponent = false)
```

Comment:
Wondering if we should rename this method to TryDisableInputAction now. It's become very conditional.